content/en/docs/contribute/generate-ref-docs/contribute-upstream.md

```diff
@@ -53,24 +53,34 @@ creating a patch to fix it in the upstream project.
 
 ## Cloning the Kubernetes repository
 
-If you don't already have the kubernetes/kubernetes repository, get it now:
+If you don't already have the kubernetes/kubernetes repository, get it now, and if you already have a fork of `kubernetes` then make sure your `kubernetes` fork is up-to-date with the `kubernetes/kubernetes` remote on
+GitHub (`master` branch), and clone your `kubernetes` fork.
 
 ```shell
 mkdir $GOPATH/src
 cd $GOPATH/src
-go get github.com/kubernetes/kubernetes
+git clone git@github.com:<your_github_username>/kubernetes.git
 ```
 
 Determine the base directory of your clone of the
 [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes) repository.
 For example, if you followed the preceding step to get the repository, your
-base directory is `$GOPATH/src/github.com/kubernetes/kubernetes.`
+base directory is `$GOPATH/src/kubernetes`.
 The remaining steps refer to your base directory as `<k8s-base>`.
 
+Also, create a fork of kubernetes/reference-docs and if you already have a fork then 
+make sure your `reference-docs` fork is up-to-date with the `kubernetes-sigs/reference-docs` remote on Github (`main` branch), and clone your `reference-docs` fork.
+
+```shell
+mkdir $GOPATH/src
+cd $GOPATH/src
+git clone git@github.com:<your_github_username>/reference-docs.git
+```
+
 Determine the base directory of your clone of the
 [kubernetes-sigs/reference-docs](https://github.com/kubernetes-sigs/reference-docs) repository.
 For example, if you followed the preceding step to get the repository, your
-base directory is `$GOPATH/src/github.com/kubernetes-sigs/reference-docs.`
+base directory is `$GOPATH/src/reference-docs.`
```

Comment:
To be aligned with the earlier comment:
base directory is `$GOPATH/src/k8s.io/reference-docs.`